src/hooks/useDragAndDropToolbarButton.ts

```diff
@@ -19,46 +18,21 @@ import { shortcutById } from '../shortcuts'
 import store from '../stores/app'
 import appendToPath from '../util/appendToPath'
 
-export type DraggableToolbarButtonProps = ToolbarButtonProps & {
+type DraggableToolbarButtonProps = {
   path: Path
   simplePath: SimplePath
```

Comment:
`path` and `simplePath` are not used by `useDragAndDropToolbarButton` and should be removed.